examples/plot_knockoff_aggregation.py

```diff
@@ -37,70 +35,80 @@
 
 
-def single_run(
-    n_subjects, n_clusters, rho, sparsity, fdr, n_bootstraps, n_jobs, seed=None
-):
+#######################################################################
+# Define the function for running the three procedures on the same data
+# ---------------------------------------------------------------------
+def single_run(n_samples, n_features, rho, sparsity, snr, fdr, n_bootstraps, seed=None):
     # Generate data
     X, y, _, non_zero_index = multivariate_1D_simulation_AR(
-        n_subjects, n_clusters, rho=rho, sparsity=sparsity, seed=seed
+        n_samples, n_features, rho=rho, sparsity=sparsity, seed=seed, snr=snr
     )
 
     # Use model-X Knockoffs [1]
-    selected, test_scores, threshold, X_tildes = model_x_knockoff(
-        X,
-        y,
-        estimator=LassoCV(
-            n_jobs=n_jobs,
-            verbose=0,
-            max_iter=1000,
-            cv=KFold(n_splits=5, shuffle=True, random_state=0),
-            tol=1e-6,
-        ),
-        n_bootstraps=1,
-        random_state=seed,
-    )
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=ConvergenceWarning)
+        warnings.filterwarnings("ignore", category=UserWarning, module="hidimstat")
+        selected, test_scores, threshold, X_tildes = model_x_knockoff(
+            X,
+            y,
+            estimator=LassoCV(
+                n_jobs=1,
+                cv=KFold(n_splits=5, shuffle=True, random_state=0),
+            ),
+            n_bootstraps=1,
+            random_state=seed,
+        )
     mx_selection, _ = model_x_knockoff_pvalue(test_scores, fdr=fdr)
     fdp_mx, power_mx = fdp_power(mx_selection, non_zero_index)
 
+    # Use aggregation model-X Knockoffs [2]
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=ConvergenceWarning)
+        warnings.filterwarnings("ignore", category=UserWarning, module="hidimstat")
+        selected, test_scores, threshold, X_tildes = model_x_knockoff(
+            X,
+            y,
+            estimator=LassoCV(
+                n_jobs=1,
+                cv=KFold(n_splits=5, shuffle=True, random_state=0),
+            ),
+            n_bootstraps=n_bootstraps,
+            n_jobs=1,
+            random_state=seed,
+        )
+
     # Use p-values aggregation [2]
-    selected, test_scores, threshold, X_tildes = model_x_knockoff(
-        X,
-        y,
-        estimator=LassoCV(
-            n_jobs=n_jobs,
-            verbose=0,
-            max_iter=1000,
-            cv=KFold(n_splits=5, shuffle=True, random_state=0),
-            tol=1e-6,
-        ),
-        n_bootstraps=n_bootstraps,
-        n_jobs=n_jobs,
-        random_state=seed,
-    )
     aggregated_ko_selection, _, _ = model_x_knockoff_bootstrap_quantile(
-        test_scores, fdr=fdr, gamma=0.3
+        test_scores, fdr=fdr, adaptive_aggregation=True
     )
 
     fdp_pval, power_pval = fdp_power(aggregated_ko_selection, non_zero_index)
```

Comment:
citation for e-values (L116) should be [3]

Reply:
done

Reply:
could you also check L116 ?

Reply:
done